mkosi/resources/mkosi-tools/mkosi.conf.d/10-debian-kali-ubuntu/mkosi.conf.d/python3-virt-firmware.conf

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[TriggerMatch]
+Distribution=debian
+Release=!bookworm
```

Comment:
If you'd be willing to configure backports for the tools tree, bookworm would work as well.

Reply:
No point I think since the tools tree release will almost always be debian testing anyway, the primary reason we do these conditionalizations is for `mkosi dependencies` where we can't enable backports